apiExamples/swapValue.html

```diff
@@ -0,0 +1,7 @@
+<auro-input id="swapExampleLeft" bordered>
+  <span slot="label">Left Input</span>
+</auro-input>
+<auro-button id="swapExampleBtn">Swap Values</auro-button>
```

Comment:
When it comes to the button, I don't see why we simply can't do something like the following:

```html
<button>
   <svg viewBox="0 0 24 24" width="24px" height="24px" 

           aria-hidden="true" focusable="false">
        <!-- svg content -->
    </svg>
    <span class="visually-hidden">Swap input values</span>

</button>
```

Icons are already set to be a specific height that works with all designs and we just need to ensure that we are accessible. There is no dependency on any other kind of component update that I can see necessary.

Reply:
Resolving without direct change due to review comments being added to a draft PR that was not ready for review.